scripts/ci/azure/build_visit_develop

```diff
@@ -1,16 +1,17 @@
 #!/bin/bash
-export VISIT_VERSION=${VISIT_VERSION:-"3.1.4"}
+export VISIT_VERSION=${VISIT_VERSION:-"3.2.0"}
 export TRUNK_BUILD="yes"
```

Comment:
Given that this file was so out-dated, it makes me wonder if it is even being used anymore?

Reply:
Agreed...

Reply:
@brugger1 do you have any insights on this?

Reply:
I believe this would have been used to build the CI container, but masonry is now being used to build the container, so I agree it is not used.

Reply:
Great. I'll go ahead and remove this file then.